src/ewasm/codegen/imports.rs

```diff
@@ -0,0 +1,166 @@
+extern crate inkwell;
+
+use crate::ewasm::codegen::Codegen;
+use inkwell::attributes::{Attribute, AttributeLoc};
+use inkwell::module::Linkage;
+use inkwell::types::{BasicType, FunctionType};
+use inkwell::values::BasicValue;
+use inkwell::AddressSpace;
+
+// For now this will just set up what will often likely need, i.e. getCaller and revert
+impl<'a, 'ctx> Codegen<'a, 'ctx> {
+    pub fn ether_imports(&self) {
+        // self.import_get_caller(); // Using a dummy getCaller while we are testing TODO remove for release
```

Comment:
Should this stay in?

Reply:
It should stay in until we release a binary. We require the dummy implementation for all caller protections tests